Automattic/wp-calypso
# Remove Jetpack Search Coupon Injection

#### Changes proposed in this Pull Request

* Remove custom coupon code injection

#### Testing instructions

1. Boot branch as Calypso Blue, navigate to plans for a Jetpack Site
2. Add Jetpack Search to a cart
3. Verify there is no error notification, and that the discount in the cart is not provided via a coupon code.
 
<img width="582" alt="Screen Shot 2022-03-01 at 12 47 03" src="https://user-images.githubusercontent.com/2810519/156246379-5ef5f97a-86b9-4b08-b78a-d937fd873cc4.png">
